ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -512,6 +546,13 @@ public boolean onEditorAction(TextView v, int actionId, KeyEvent keyEvent) {
                       SystemClock.nanoTime(),
                       editText.getText().toString()));
             }
+            if (actionId == EditorInfo.IME_ACTION_NEXT ||
```

Comment:
Special handling allows us to use PREVIOUS/NEXT. Without that, Android will try to call the default behavior on these actions (moving to the previous/next focusable view), which was way too unstable for React Native.